### DISCUSSION TITLE
Help with montage command

### DISCUSSION THREAD

**Discussion by Flyguy224** [ImageMagick/ImageMagick#4680] [Help]
I have a folder of images that I would like to be combined into one large image with no space in between each image and having 100 images across with however many columns it takes while retaining the original dimensions of the small images (288widthx120height). I've never used Command Line before and I'm not understanding it. The thing is, using ImageMagick would speed things up immeasurably, because I have a ton of images and putting them all into Photoshop one by one is maddening.

**Comment by fmw42** [comment]
Assume all images are the same dimensions, otherwise there will be some space between some of them. Change directories to your folder

`montage * -tile 100x -geometry +0+0 result.suffix`

change suffix to PNG or JPG or whatever output format you want.

**Reply by Flyguy224** [reply]
How would I get it to go chronologically by filename? Mine are all numbers. Thank you, something like this is _exactly_ what I'm looking for.

**Reply by fmw42** [reply]
Please show a short list of the files in your directory.  Files are always listed from the wild card * alphabetically.

**Comment by Flyguy224** [comment]
Oh, I got it. Thank you so much for your help! You are the best.

**Comment by Flyguy224** [comment]
Sigh. No I don't have it. Currently, the files are named strictly so they
go in the order I want them. Naming conventions aren't really important to
me. So currently, I have

A (1)
A (2)
A (3)
A (4)
A (5)
....
etc.

**Comment by snibgo** [comment]
> ...  100 images across ...
> Currently, the files are named strictly so they go in the order I want them. ... So currently, I have
A (1)
A (2)

That won't work, because lexicographically (ie "in dictionary order"), "A (2)" comes after "A (12)" etc. You need leading zeroes: "A (002)" etc.

**Comment by Flyguy224** [comment]
Oh, I see. Thank you. I believe I can do it now.
